src/index.js

```diff
@@ -0,0 +1,233 @@
+devENV = false
+const baseURL = devENV ? 'http://localhost:5000' : 'http://christellenkera.com/weather-report-api'
+
+"use strict";
+// Common Selectors
+// const sky = document.getElementById("sky");
```

Comment:
Remove unused code that is commented out. It could accidentally get commented back in and execute when it shouldn't, but also calls into question "why is this commented out? is it needed?" for other devs that come along.